packages/expo-passkeys/src/utils.ts

```diff
@@ -1,6 +1,5 @@
-import { Buffer } from 'node:buffer';
-
 import { ClerkWebAuthnError } from '@clerk/shared/error';
+import { Buffer } from 'buffer';
```

Comment:
We might want to use an ESLint rule in this package to ensure we don't use the `node:` protocol